codegen/src/main/java/software/amazon/awssdk/codegen/model/config/customization/CustomizationConfig.java

```diff
@@ -131,6 +131,11 @@
 
     private boolean useAutoConstructList = true;
 
+    /**
+     * Custom Retry Policy
+     */
+    private String customRetryPolicy;
```

Comment:
Why do we need retry policy in customization config? I would expect it in some kind of configuration class that is set on the client.

Reply:
By default, all clients builder will use `AwsRetryPolicy.DEFAULT` policy. https://github.com/aws/aws-sdk-java-v2/pull/546/files#diff-f5432db5535088f2b387cb75be778781R106

We need it in codegen so that we can add it in the generated service DefaultBaseClientBuilder.

For example, In `DefaultDynamoDBBaseClientBuilder` class, service specific retry policy is applied in `mergeServiceDefaults`.

```java
    @Override
    protected final SdkClientConfiguration mergeServiceDefaults(SdkClientConfiguration config) {
        return config.merge(c -> c.option(SdkAdvancedClientOption.SIGNER, defaultSigner())
                .option(SdkClientOption.CRC32_FROM_COMPRESSED_DATA_ENABLED, true)
                .option(SdkClientOption.RETRY_POLICY, DynamoDbRetryPolicy.defaultPolicy()));
    }
```